docs/team/icytornado.md

```diff
@@ -5,16 +5,55 @@ title: Zhang Peng's Project Portfolio Page
 
 ## Project: App-Ointment
 
-App-Ointment is a patient appointment management software adapted from AddressBook - Level 3, which is a desktop address book application used for teaching Software Engineering principles. The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java, and has about 10 kLoC.
+App-Ointment is a patient appointment management software adapted from AddressBook - Level 3, which is a desktop address book application used for teaching Software Engineering principles. The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java, and has about 19 kLoC.
 
 Given below are my contributions to the project.
 
-* **Code contributed**:
+* **Code contributed**: [RepoSense] (https://nus-cs2103-ay2021s2.github.io/tp-dashboard/#breakdown=true&search=icytornado)
+
+* **Features Implemented**:
+  * Implementation of the `edit-appt` feature:
+    - *Description*:
+        - `edit-appt` command allow the user to edit an existing appointment by changing the fields for the appointment.
+        - `edit-appt` + `index` + `field` to edit => edit the specified field of the appointment specified by the appointment index.
+        - the `field` to edit is preceded by its related prefix, i.e. `pt/` for patient, `dr/` for doctor, `at/` for appointment start date and time, `to/` for appointment end date and time, `dur/` for appointment duration, `/t` for appointment tags.
+    
+  * Implementation of the `Doctor` feature
+    - *Description*:
+        - Created the `Doctor` class and implemented its related command features including
+          `delete-doctor`, `edit-doctor`, `add-doctor`, `clear-doctor`, `list-doctor`.
+            
+  * Added some relevant test classes for `Doctor`.
```

Comment:
i think it is only fair to mention here that there were a lot of issues with the implementation, and other members of the team had to step in to resolve those issues.

Reply:
i think i just say what i added in.